cmd/juju/application/addunit.go

```diff
@@ -24,32 +24,42 @@ var usageAddUnitSummary = `
 Adds one or more units to a deployed application.`[1:]
 
 var usageAddUnitDetails = `
-Adding units to an existing application is a way to scale out that application. 
-Many charms will seamlessly support horizontal scaling, others may need an
-additional application to facilitate load-balancing (check the individual 
-charm documentation).
-This command is applied to applications that have already been deployed.
-By default, applications are deployed to newly provisioned machines in
-accordance with any application or model constraints. Alternatively, this 
-command also supports the placement directive ("--to") for targeting
-specific machines or containers, which will bypass any existing
-constraints.
+Adding units to an existing application is a way to scale out that
+application.
+
+Many charms will seamlessly support horizontal scaling while others
+may need an additional application support to facilitate replication
+or load-balancing. See the documentation for specfic charms to check
+how scale-out is supported.
+
+The add-unit is applied to applications that have already been
+deployed.  By default, applications are deployed to newly provisioned
+machines in accordance with any application or model
+constraints. Alternatively, this command also supports the placement
+directive ("--to") for targeting specific machines or containers,
+which will bypass any existing constraints.
 
 Examples:
-Add five units of wordpress on five new machines:
 
+Add five units of wordpress on five new machines:
     juju add-unit wordpress -n 5
 
-Add one unit of mysql to the existing machine 23:
-
+Add a unit of mysql to an existing machine:
     juju add-unit mysql --to 23
 
-Create a new LXD container on machine 7 and add one unit of mysql:
+Add two units of mysql to two preexisting machines:
```

Comment:
That's interesting and weird. With my rewording the word "preëxisting" no longer exists in the text :)